std/conv.d

```diff
@@ -207,6 +207,13 @@ template to(T)
     {
         return toImpl!T(arg);
     }
+
+    // Fix issue 16108
+    T to(S)(ref S arg)
+        if (isAggregateType!S && !isCopyable!S)
+    {
+        return toImpl!T(arg);
+    }
 }
```

Comment:
No because then you would be modifying the behavior of the function for several types, most notably ranges.